permissions/plugin-nowsecure-auto-security-test.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: "nowsecure-auto-security-test"
+github: "jenkinsci/nowsecure-auto-security-test-plugin"
+paths:
+- "org/jenkins-ci/plugins/nowsecure"
```

Comment:
This does not correspond to groupId + artifactId; please see README.md in this repo.

Reply:
@daniel-beck , can you review it again. I removed nowsecure from the name for this plugin.